src/sentry/snuba/models.py

```diff
@@ -144,3 +151,16 @@ def write_relocation_import(
         subscription.save()
 
         return (subscription.pk, ImportKind.Inserted)
+
+
+@data_source_type_registry.register("snuba_query_subscription")
```

Comment:
🤔 is it too much overkill to export this string as an enum where we define the registry? i don't think it adds a lot of value and kinda detracts from the simplicity of the registry, but it seems like it might be nice since we're re-using this string a couple of times and reduces chances of a typo breaking things.

Reply:
I was mostly planning on us never using the strings - they're just references in the database for loading the handlers. You can reverse lookup via `data_source_type_registry.get_key(QuerySubscriptionDataSourceHandler)` in tests. I don't think we ever need to reference the strings in actual production code otherwise... if we do, then an enum could make sense